compiler/data/function-data.cpp

```diff
@@ -267,6 +267,8 @@ string FunctionData::get_human_readable_name(bool add_details) const {
   std::string result_name;
   if (is_lambda()) {
     result_name = get_params().empty() ? "anonymous()" : "anonymous(...)";
+  } else if(type == FunctionData::func_main) {
+    return "todo";//todo
```

Comment:
Are we going to make readable names for the `main` functions in this PR or put them in a separate one?

Reply:
I decided to do it later, as it has no relation to colors, what this PR is about